global.php

```diff
@@ -647,8 +647,7 @@
 			}
 			else
 			{
-				$total_joinrequests = my_number_format($total_joinrequests);
-				$lang->pending_joinrequests = $lang->sprintf($lang->pending_joinrequests, $total_joinrequests);
+				$lang->pending_joinrequests = $lang->sprintf($lang->pending_joinrequests, my_number_format($total_joinrequests));
```

Comment:
Why this change?

Reply:
Maybe I should revert it back if it is necessary. I wanted to solve the mentioned issue in a different way but then I found easier way without a lot of changes... this should be fine, but no problem to change it to the default code.

Reply:
I don't mind, but it would probably be better to keep has it was? @mybb/developers

Reply:
If it's no longer being used to resolve the issue we should probably revert this part.